spring-cloud-starter-common-security-config-web/src/test/java/org/springframework/cloud/common/security/SpringCloudCommonSecurityApplicationTests.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2016-2022 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.springframework.cloud.common.security;
+
+import org.junit.jupiter.api.Test;
+
+import org.springframework.boot.test.context.SpringBootTest;
+
+/**
+ * Testing startup and configuration
+ *
+ * @author Corneil du Plessis
+ */
+@SpringBootTest
+class SpringCloudCommonSecurityApplicationTests {
+
+    @Test
+    void contextLoads() {
+    }
```

Comment:
Is there anything expected to be in the context that we should sanity check - or is that already done elsewhere?

Reply:
Starting up the application is enough to trigger the cycle and verify configuration.